packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -78,9 +76,15 @@ const SupportListItem = ({ item }: { item: SupportItem }) => {
   );
 };
 
-export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
-  const { items } = useSupportConfig();
+declare type SupportButtonProps = {
+  supporters?: SupportItem[];
```

Comment:
The name `supporters` seems a bit odd. What about `supportItems` or maybe just `items`?

Reply:
Another option would be to just accept `<SupportListItem/>` as children directly rather then adding a prop for this. As a rough example.

```tsx
<SupportButton>
  <span>Some heading text</span>
  <SupportListItem ...propsHere />
  <SupportListItem ...propsHere />
</SupportButton>
```

Although, thinking about it, maybe the `SupportButton` should also have a proper `title` prop for the main heading which is currently the `children`. That way it could be styled consistently. Although that would be a breaking change.